config/locales/cz.yml

```diff
@@ -225,184 +226,180 @@ cz:
 
       mailers:
         edit_subscription:
-          subject: Request for Super Secret Support Link
-          header: Your Super Secret Support Link has arrived!
-          link_description: Anyone with the above link can use it to make changes to your subscription. It will expire in one hour. Keep it secret, keep it safe.
-          repeat_request: If you don't get around to making the changes you want in time, no worries. We can always send you another super secret link.
-          more_help: You can also reply to this email if need more assistance.
+          subject: Požádej o svů supertajný link na podporu
+          header:  Tvůj supertajný kod dorazil!
+          link_description: Kdokoliv kdo má přístup k odkazu nahoře, ho může použít k tomu aby změnil částku tvého předplatného. Za hodinu vyprší, udržuj ho v tajnosti a bezpečí
+          repeat_request: Jestli nestihneš co si potřeboval, není se čeho bát, vždy ti můžeme poslad další odkaz
+          more_help: Můžeš taky odpovědět na tento email jestli potřebuješ s něčím pomoct.
 
     languages:
-      view_site_in_locale:  View this site in Czech [here](https://cs.crimethinc.com).
-      view_tools_in_locale: You can access an array of posters, zines, and other materials in Czech [here](https://cs.crimethinc.com/tools).
-      view_books_in_locale: You can access our books in Czech [here](https://cs.crimethinc.com/books).
+      view_site_in_locale:  Podívej se na tuto stránku v angličtině [zde](https://crimethinc.com).
+      view_tools_in_locale: Tady najdeš řadu plakátů, zinů, a dalších materiálů v Angličtině  [zde](https://crimethinc.com/tools).
+      view_books_in_locale: Přístup k našim knihám v angličtině [zde](https://crimethinc.com/books).
 
-      heading: Languages
+      heading: Jazyky
       description: |
-        CrimethInc. material has appeared in over three dozen languages; our introductory
-        primer, [To Change Everything](/tce), is available in 30. We have
-        [long](/2010/08/18/non-english-crimethinc-projects) aspired to collect all the different
-        translations on this site. If you would like to translate our material—or if you can
-        help us to collect translations for this archive or help us maintain communication
-        between related projects around the world—please
-        [contact us](mailto:foreignlegion@crimethinc.com).
-
-        Here, you can find all the articles on this site categorized according to language.
+        CrimethInc. materiály se objevily ve více než třech desítkách jazyků; náš úvodní text, [Změňit vše](/tce), je dostupný ve třiceti.
+        [Dlouho](/2010/08/18/non-english-crimethinc-projects) se snažíme přeložit tuto stránku. Jestli by si chtěl přeložit náš materiál—nebo jestli nám můžeš pomoct najít překlady, nebo nám můžes pomoct v udržení komunikace s příbuznými projekty na celém.
```

Comment:
The "we have" in the english version seems to have been dropped here between "třiceti" and "Dlouho"

Reply:
@astronaut-wannabe  "We have" isn't necessary here  the "we" part is included in "snažíme" which could be translated into english like "we are trying"  If that's what you're refering to

Reply:
sounds good to me!